src/Resources/Workflow.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace Novu\SDK\Resources;
+
+class Workflow extends Resource
+{
+    /**
+     * @var string
+     */
+    public $id;
+
```

Comment:
Please check the docblooks for the other resources in the repo. There should also be a description for what the variable represents.